samples/exemplar/src/main/kotlin/com/squareup/exemplar/dashboard/admin/AlphaIndexAction.kt

```diff
@@ -0,0 +1,38 @@
+package com.squareup.exemplar.dashboard.admin
+
+import kotlinx.html.div
+import kotlinx.html.h1
+import misk.scope.ActionScoped
+import misk.web.Get
+import misk.web.HttpCall
+import misk.web.ResponseContentType
+import misk.web.actions.WebAction
+import misk.web.dashboard.AdminDashboardAccess
+import misk.web.mediatype.MediaTypes
+import misk.web.v2.DashboardIndexAction.Companion.titlecase
+import misk.web.v2.DashboardPageBuilder
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class AlphaIndexAction @Inject constructor(
+  @JvmSuppressWildcards private val clientHttpCall: ActionScoped<HttpCall>,
+  private val dashboardPageBuilder: DashboardPageBuilder,
+) : WebAction {
+  @Get(PATH)
+  @ResponseContentType(MediaTypes.TEXT_HTML)
+  @AdminDashboardAccess
+  fun get(): String = dashboardPageBuilder
+    .newBuilder()
+    .title { appName, dashboardHomeUrl, dashboardTab -> "${dashboardTab?.name?.let { "$it | " } ?: ""}${dashboardTab?.category} on $appName ${dashboardHomeUrl?.dashboardAnnotationKClass?.titlecase() ?: ""}" }
+    .httpCall(clientHttpCall.get())
```

Comment:
why do we need to pass the `HttpCall` to it?

Reply:
It's a convenience method to get the path instead of calling `clientHttpCall.get().url.encodedPath` at each callsite.

Reply:
I moved the call injection into the class which seems to be working well and make the callsite simpler